SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,73 @@
+    |                  | Greater than                            | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Greater or Equal than                   | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less than                               | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less or Equal than                      | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | XOR                                     | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Exponential                             | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Log                                     | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Max                                     | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Reciprocal                              | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Relu                                    | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Sigmoid                                 | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Softmax                                 | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Tanh                                    | - FSS        |                     2 |
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | Public           | Multiplication                          | - FSS        | - 2                   |
```

Comment:
In addition, subtraction also has public support with Falcon.

Reply:
You are right. The first part of the table, the visibility is **Public & Private**. I did not want to repeat all the operators, so I guessed it would be nice to separate between **Public & Private**, **Private** and **Public**. So, I think I already expressed the addition and subtraction support for Falcon correctly. If you are okey, you can resolve this conversation. Otherwise, ping me again!